stock_request/security/ir.model.access.csv

```diff
@@ -15,3 +15,4 @@ access_stock_request_user_warehouse,stock request user Warehouse,stock.model_sto
 access_stock_request_manager_warehouse,stock request manager Warehouse,stock.model_stock_warehouse,group_stock_request_manager,1,0,0,0
 access_stock_request_user_stock_move,stock request user stock move,stock.model_stock_move,group_stock_request_user,1,0,0,0
 access_stock_request_manager_stock_move,stock request manager stock move,stock.model_stock_move,group_stock_request_manager,1,0,0,0
+access_stock_warehouse_orderpoint,sotck request user stock warehouse orderpoint,stock.model_stock_warehouse_orderpoint,group_stock_request_user,1,0,0,0
```

Comment:
This is not required to manage stock_request in a default way. What do you think @etobella? 
If you keep this please correct the typo on 'sotck'

Thanks!

Reply:
I added this rule to pass the tests when the tests try to confirm a picking with stock_request_user it raises a permission error.

```
Traceback (most recent call last):
  File "/Users/Ramitos/Github/14.0/odoo/odoo/tools/cache.py", line 85, in lookup
    r = d[key]
  File "/Users/Ramitos/Github/14.0/odoo/odoo/tools/func.py", line 71, in wrapper
    return func(self, *args, **kwargs)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/tools/lru.py", line 34, in __getitem__
    a = self.d[obj]
KeyError: ('ir.model.access', <function IrModelAccess.check at 0x105324170>, 10, False, 'stock.warehouse.orderpoint', 'read', True, (None,))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/Ramitos/Github/14.0/stock-logistics-warehouse/stock_request/tests/test_stock_request.py", line 737, in test_cancel_request
    picking.with_user(self.stock_request_user).action_confirm()
  File "/Users/Ramitos/Github/14.0/odoo/addons/stock/models/stock_picking.py", line 700, in action_confirm
    self.mapped('move_lines').filtered(lambda move: move.state not in ('draft', 'cancel', 'done'))._trigger_scheduler()
  File "/Users/Ramitos/Github/14.0/odoo/addons/stock/models/stock_move.py", line 1741, in _trigger_scheduler
    ], limit=1)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/models.py", line 1697, in search
    res = self._search(args, offset=offset, limit=limit, order=order, count=count)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/models.py", line 4444, in _search
    model.check_access_rights('read')
  File "/Users/Ramitos/Github/14.0/odoo/odoo/models.py", line 3312, in check_access_rights
    return self.env['ir.model.access'].check(self._name, operation, raise_exception)
  File "<decorator-gen-34>", line 2, in check
  File "/Users/Ramitos/Github/14.0/odoo/odoo/tools/cache.py", line 90, in lookup
    value = d[key] = self.method(*args, **kwargs)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/addons/base/models/ir_model.py", line 1758, in check
    raise AccessError(msg)
odoo.exceptions.AccessError: You are not allowed to access 'Minimum Inventory Rule' (stock.warehouse.orderpoint) records.

```

Reply:
This problem seems to be related with some change in demo data that Odoo applied on v14. Maybe identifying the product that is leading the problem and change by other can make the fix. Apart of that, I have no inconvenient of adding this access right because order-point model is on stock module.